README.md

```diff
@@ -39,6 +40,8 @@ module "static_site" {
 * `hosted_zone_id` - The hosted zone matching your domain; if provided, the module will create the Route53 recordset for you.  If not, you'll need to map the cloudfront DNS yourself
 * `hsts_header` - The value of the HSTS header, eg `max-age=31536000`, helpful to let your users know you always want them using HTTPS.  With CloudFront, you need lambda@edge to do this, so... it will create it.
 * `default_subdirectory_object` - If you want URLs ending in `/` to load a file, set this to something like `index.html`. With CloudFront, you need lambda@edge to do this, so... it will create it.
+* `content_bucket_name` - Set the name of the content bucket.  Defaults to `<domain_name>-static-content`
+* `create_content_bucket` - Set whether module creates the bucket, or looks it up with a data-source.  Defaults to `true`
```

Comment:
Already discussed this with you but wanted to get this in the PR itself. This is arguably out of scope for this ticket BUT as long as we are doing this we should also make the creation of a cloudfront distribution configurable.

I personally think we should put this PR on hold and create a new ticket to do this work.

Reply:
Chatted with the gang and decided I am wrong about this.